src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/orderDetails.html.twig

```diff
@@ -4,7 +4,7 @@
     {% set lastNewPayment = order.getLastNewPayment() %}
     {% include '@SyliusShop/Common/Order/_summary.html.twig' %}
 
-    {% if order.paymentState is not same as ('paid') %}
+    {% if order.paymentState is not same as ('paid') and order.paymentState is not same as ('cancelled') %}
```

Comment:
Side note: we should have issers for paid / cancelled / etc. states. The current implementation depends heavily on the state machine, but not on the model. Then we can do `{% if not order.paid and not order.cancelled %}`.